daemon/graphdriver/btrfs/btrfs.go

```diff
@@ -51,29 +53,35 @@ func Init(home string, options []string) (graphdriver.Driver, error) {
 	return graphdriver.NaiveDiffDriver(driver), nil
 }
 
+// Driver contains information about the filesystem mounted.
 type Driver struct {
+	//root of the file system
 	home string
 }
 
+// String prints the name of the driver (btrfs).
 func (d *Driver) String() string {
 	return "btrfs"
 }
 
+// Status returns current driver information.
```

Comment:
@brahmaroutu The point of godoc is to provide a lost developer with a hint about the value using a particular method. With stringly typed data on an implementation, documenting the return value is completely valid. In this case, the method always returns the same values, since it is specific to the btrfs driver.

A good approach might be to give guidance on the meaning of "Build Version" and "Library Version" and how they can be used. Are they simply informational or can they be used to infer supported features?